### ISSUE TITLE
Library Upgrades for Plone 7

### ISSUE THREAD

**Issue by tisto** [plone/volto#6327]

**Comment by Shyam-Raghuwanshi** [comment]
hi @sneridagh would you like to guide me for this issue

**Comment by stevepiercy** [comment]
@Shyam-Raghuwanshi no. Please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).
